Account.java

```diff
@@ -4,14 +4,78 @@
 	
 	private String accountName;
 	private double openingBalance;
+	private double currentBalance;
+	private double amount;
+	private String payee;
+
+
+	//constructor
+	public Account() {
+
+	}
+
 
 	public Account(String accountName, double openingBalance) {
 		this.accountName = accountName;
 		this.openingBalance = openingBalance;
 	}
-	
+
+
+
 	public String toString() {
 		return (accountName + ": " + openingBalance);
 	}
 
+
+	//account = new Account("Main", 100); // account object of the class Account
+
+	// internal transfer out of account, update balance
+	public void internalTransferOut(double Amount, String accountName){
+		this.amount = Amount;
+		this.accountName = accountName;
+		currentBalance = openingBalance - amount;
+	}
+
+	// internal transfer into account, update balance
+	public void internalTransferIn(double Amount, String accountName){
+		this.amount = Amount;
+		this.accountName = accountName;
+		currentBalance = openingBalance + amount;
+	}
+
+
+	public double Transfer(double Amount, String From, String To){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.